dspace/modules/journal-submit/src/main/java/org/datadryad/submission/DryadEmailSubmission.java

```diff
@@ -351,30 +274,33 @@ else if (property.equals("fullname")) {
             }
         }
 
-        if (LOGGER.isDebugEnabled()) {
-            LOGGER.debug("Checking that all journals are correctly registered");
-        }
+        LOGGER.debug("Checking that all journals are correctly registered");
 
         // Returns validated map or throws an exception if there are problems
         myJournals = validate(journals);
         // Returns a mapping of Journal Names to Journal Codes
         myJournalNames = mapJournalNamesToCodes(journals);
+
+        LOGGER.debug("scheduling email harvesting");
+        myEmailHarvester = new Timer();
+        // schedule email harvesting to happen once a day
+        myEmailHarvester.schedule(new DryadEmailSubmissionHarvester(), 0, 1000 * 60 * 60 * 24);
```

Comment:
Once a day is probably not frequent enough. The submitter will have the link to submit their data as soon as we do, so we probably don't want to introduce a delay of up to 24h. This should be configurable.
Also, having an on-demand option would be good too, . The current process is instantaneous, and when working through setting up new journals, I think that tight feedback loop helps.